javascript/utils/MaplibreStyles.d.ts

```diff
@@ -556,14 +553,11 @@ export interface LineLayerStyleProps {
    *
    * @requires lineTranslate
    */
-  lineTranslateAnchor?: Value<
-    Enum<LineTranslateAnchorEnum, LineTranslateAnchorEnumValues>,
-    ["zoom"]
-  >;
+  lineTranslateAnchor?: "map" | "viewport";
   /**
    * Stroke thickness.
    */
-  lineWidth?: Value<number, ["zoom", "feature", "feature-state"]>;
+  lineWidth?: number;
```

Comment:
Did you notice this breaks a lot of expression styles? @tyrauber @caspg

Reply:
I did not.  This file is programmatically generated from the published style-spec. So these changes were made upstream.  I was able to build for both iOS and Android, and our test suite failed to detect any issues.  Can you give me specifics.  I can revert this change, but the Android SDK requires an up-to-date style-spec.